app/common/containers/pages/MedicationRequestsListPage/index.js

```diff
@@ -62,6 +66,17 @@ const MedicationRequestsListPage = ({
         onSubmit={values => setFilter(values, { location, router })}
       />
     </div>
+    <div>
+      <DateFilterForm
+        initialValues={
+          activeDateFilter.length === 2 && {
+            created_from: location.query[activeDateFilter[0]],
+            created_to: location.query[activeDateFilter[1]]
+          }
+        }
+        onSubmit={values => setFilter(values, { location, router })}
```

Comment:
@kiurchv need your comments here

Reply:
Assuming that we getting filter from the `getFilter` function from previous comment we can set initial values in that way:
```jsx
<DateFilterForm
  initialValues={activeDateFilter.reduce(
    (filter, name) => ({ ...filter, [name]: location.query[name] }),
    {}
  )}
  onSubmit={values => setFilter(values, { location, router })}
/>
```